packages/repository-tests/src/crud/relations/acceptance/belongs-to-inclusion-resolver.relation.acceptance.ts

```diff
@@ -0,0 +1,221 @@
+            zipcode: '8200',
+            customerId: thor.id,
+          });
+          const result = await addressRepo.find({
+            include: [{relation: 'customer'}],
+          });
+
+          const expected = {
+            ...address,
+            customer: {
+              id: thor.id,
+              name: 'Thor',
+              parentId: features.emptyValue,
+            },
+          };
+          expect(toJSON(result)).to.deepEqual([toJSON(expected)]);
+        });
+
+        it('returns related instances to target models via find() method', async () => {
+          const thor = await customerRepo.create({name: 'Thor'});
+          const odin = await customerRepo.create({name: 'Odin'});
+          const addr1 = await addressRepo.create({
+            street: 'home of Thor Rd.',
+            city: 'Thrudheim',
+            province: 'Asgard',
+            zipcode: '999',
+            customerId: thor.id,
+          });
+          const addr2 = await addressRepo.create({
+            street: 'home of Odin Rd.',
+            city: 'Valhalla',
+            province: 'Asgard',
+            zipcode: '000',
+            customerId: odin.id,
+          });
+
+          const result = await addressRepo.find({
+            include: [{relation: 'customer'}],
+          });
+
+          const expected = [
+            {
+              ...addr1,
+              customer: {
+                id: thor.id,
+                name: 'Thor',
+                parentId: features.emptyValue,
+              },
+            },
+            {
+              ...addr2,
+              customer: {
+                id: odin.id,
+                name: 'Odin',
+                parentId: features.emptyValue,
+              },
+            },
+          ];
+          expect(toJSON(result)).to.deepEqual(toJSON(expected));
+        });
+
+        it('returns related instances to target models via findById() method', async () => {
+          const thor = await customerRepo.create({name: 'Thor'});
+          const odin = await customerRepo.create({name: 'Odin'});
+          await addressRepo.create({
+            street: 'home of Thor Rd.',
+            city: 'Thrudheim',
+            province: 'Asgard',
+            zipcode: '999',
+            customerId: thor.id,
+          });
+          const addr2 = await addressRepo.create({
+            street: 'home of Odin Rd.',
+            city: 'Valhalla',
+            province: 'Asgard',
+            zipcode: '000',
+            customerId: odin.id,
+          });
+
+          const result = await addressRepo.findById(addr2.id, {
+            include: [{relation: 'customer'}],
+          });
+          const expected = {
+            ...addr2,
+            customer: {
+              id: odin.id,
+              name: 'Odin',
+              parentId: features.emptyValue,
+            },
+          };
+          expect(toJSON(result)).to.deepEqual(toJSON(expected));
+        });
+      });
+    },
+  );
+}
```

Comment:
Can you add a test for `findOne`?

Reply:
I am actually thinking about deleting `findById()`. My intention was to show more use cases instead of testing `find*` method inside of inclusion resolver acceptance tests. Thought?

Reply:
Are we testing the other `find` methods somewhere else?

Reply:
I believe there must have some unit tests in `legacy-juggler-bridge.unit.ts` for `find*`. If those work, then we shouldn't be worrying about them here. That's why I don't think it's necessary to test them all here.

Reply:
okay i'm fine with whatever you decide then